ouroboros-network/src/Ouroboros/Network/BlockFetch.hs

```diff
@@ -115,14 +114,14 @@ data BlockFetchConsensusInterface peer header block m =
        -- They must be already validated and contain the last @K@ headers
        -- (unless we're near the chain genesis of course).
        --
-       readCandidateChains    :: STM m (Map peer (ChainFragment header)),
+       readCandidateChains    :: STM m (Map peer (AnchoredFragment header)),
 
        -- | Read the K-suffix of the current chain.
        --
        -- This must contain info on the last @K@ blocks (unless we're near
        -- the chain genesis of course).
        --
-       readCurrentChain       :: STM m (ChainFragment block),
+       readCurrentChain       :: STM m (AnchoredFragment block),
```

Comment:
Can we document what assumption (or none) there is about the anchor point for the candidates vs the current chain? Are they the same anchor, or are the candidates going to be (typically) anchored on a point that is on the current chain?

Reply:
Done in the newest version.